test/integration/content-explorer/ContentExplorer.int.test.js

```diff
@@ -101,6 +101,13 @@ describe('ContentExplorer', () => {
             allowed_shared_link_access_levels: ['collaborators', 'open', 'company'],
             shared_link: null,
         });
+        cy.route('PUT', '**/files/319004423111', {
```

Comment:
We are doing one more request when clicking the share button, AND shared link does not already exist. This is the mock for that request